packages/discord.js/src/structures/Channel.js

```diff
@@ -102,19 +102,59 @@ class Channel extends Base {
   }
 
   /**
-   * Indicates whether this channel is {@link TextBasedChannels text-based}.
+   * Indicates whether this channel is a {@link TextChannel}.
    * @returns {boolean}
    */
   isText() {
-    return 'messages' in this;
+    return this.type === ChannelTypes[ChannelTypes.GUILD_TEXT];
   }
 
   /**
-   * Indicates whether this channel is {@link BaseGuildVoiceChannel voice-based}.
+   * Indicates whether this channel is a {@link DMChannel}.
+   * @returns {boolean}
+   */
+  isDM() {
+    return this.type === ChannelTypes[ChannelTypes.DM];
+  }
+
+  /**
+   * Indicates whether this channel is a {@link VoiceChannel}.
    * @returns {boolean}
    */
   isVoice() {
-    return VoiceBasedChannelTypes.includes(this.type);
+    return this.type === ChannelTypes[ChannelTypes.GUILD_VOICE];
+  }
+
+  /**
+   * Indicates whether this channel is a {@link PartialGroupDMChannel}
```

Comment:
```suggestion
   * Indicates whether this channel is a {@link PartialGroupDMChannel}.
```

Reply:
y tho

Reply:
Because you've done it for some others, and that's just really... unsettling?